text/0000-style-team.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: N/A
+- Start Date: 2022-08-25
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC charters the Rust Style Team, responsible for evolving the Rust style over time. This includes styling for new Rust constructs, as well as the evolution of the existing style over the course of Rust editions (without breaking backwards compatibility).
+
+# Motivation
+[motivation]: #motivation
+
+RFC 1607 proposed and motivated a process for determining code formatting guidelines and producing a style guide, via a temporary style team. That style guide was published as RFC 2436, and the style team wound up its operation and no longer exists. However, Rust has multiple ongoing needs for new determinations regarding Rust style, such as determining the style of new Rust constructs, and evolving the Rust style over time. Thus, this RFC re-charters the Rust Style Team as a non-temporary subteam.
+
+# Explanation and charter
+[explanation]: #explanation
+
+The renewed need for the Rust style team began to arise in discussions of language constructs such as `let`-chaining (RFC 2497) and `let`-`else` (RFC 3137). New constructs like these, by default, get ignored and not formatted by rustfmt, and subsequently need formatting added. The rustfmt team has expressed a preference to not make style determinations itself; the rustfmt team would prefer to implement style determinations made by another team.
+
+In addition, rustfmt maintains backwards compatibility guarantees: code that has been correctly formatted with rustfmt won't get formatted differently with a future version of rustfmt. This avoids churn, and avoids creating CI failures when people use rustfmt to check style in CI. However, this also prevents evolving the Rust style to take community desires into account and improve formatting over time. rustfmt provides various configuration options to change its default formatting, and many of those options represent changes that many people in the community would like enabled by default.
+
+This RFC proposes re-chartering the style team, as originally specified in RFC 1607, to determine the Rust style. This includes:
+- Making determinations about styling for new Rust constructs
+- Evolving the existing Rust style
+- Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
+
+## Team structure and membership
+
+The Rust style team will be a subteam of both the rustfmt team and the Rust language team. (For the purposes of structures that currently require a team to have only one parent team, such as the `rust-lang/team` repository, the Rust style team will have the rustfmt team as its parent team.)
```

Comment:
For my own edification, are there any other dual-parented teams within the ecosystem, and what (if any) are the differences between dual and single parented teams?

Reply:
[T-types](https://rust-lang.github.io/rfcs/3254-types-team.html#wait-it-has-two-parent-teams-can-you-do-that) is a child team of both T-lang and T-compiler.